docs/features/software-catalog/descriptor-format.md

```diff
@@ -76,7 +77,7 @@ This is the same entity as returned in JSON from the software catalog API:
     "etag": "ZjU2MWRkZWUtMmMxZS00YTZiLWFmMWMtOTE1NGNiZDdlYzNk",
     "generation": 1,
     "labels": {
-      "system": "public-websites"
+      "backstage.io/custom": "ValueStuff"
```

Comment:
Needs an update too to match what's above 😅

Reply:
Sorry about that, updated now as well